src/renderer/components/utils/Slider.tsx

```diff
@@ -54,12 +54,14 @@ class Slider extends React.Component<Props, State> {
     public componentDidUpdate() {
         if (this.state.refreshVisible) {
             this.contentElRefs.map((element, index) => {
-                const buttonList = element.getElementsByTagName("button");
-                for (const button of buttonList) {
-                    if (!this.isElementVisible(index)) {
-                        button.tabIndex = "-1";
-                    } else {
-                        button.tabIndex = "0";
+                if (element) {
```

Comment:
To help future developers / codebase reviewers, I think we need to add a code comment explaining why the `this.contentElRefs` array can contain `null` items.

My suggestion:

> The `this.contentElRefs` array is automatically populated in the `render()` > `createContent()` function, via the `div` element's `ref` callback (`ref={(ref) => this.contentElRefs[index] = ref}`), which can be invoked with `null` during the element's "unmount" lifecycle (see https://reactjs.org/docs/refs-and-the-dom.html ). Consequently, we need to check for possibly-`null` values in the `this.contentElRefs` array, in this `componentDidUpdate()` function. However, we can safely ignore usages of `this.contentElRefs` in the `moveInView()` and `isElementVisible()` functions, as these are guaranteed to be invoked when the element is still "mounted" (see the `onFocus` callback).

What do you think? Does that make sense?

Reply:
Yese, it seems to be good. I add it

Reply:
I propose also to type contentElRefs, it can be helpful

Reply:
Yes, typing would be good, BUT in this case: let's merge this critical bugfix, as we are now getting closer to a release milestone.